CoolCurrency/CurrencyViewModel/MatchCurrencyGameViewModel.swift

```diff
@@ -45,4 +47,30 @@ struct MatchCurrencyGameViewModel {
     func checkIfCorrect() -> Bool {
         selectedFlag == selectedSymbol ? true : false
     }
+    
+    var retrieveCorrectAnswer: String {
+        "\(correctAnswer) / 5"
+    }
+    
+    func shouldDisplayAnswer() -> Bool {
```

Comment:
maybe you could use an enum and switch statement here to make it more clear how the counter affects the game state?

Reply:
You can try what @justinewright suggested or cleaning up the logic a little bit will also help 😄 

```
func shouldDisplayAnswer() -> Bool { 
   counter += 1
   
   if  selectedFlag == selectedSymbol {
      correctAnswer += 1
   }
   
   if counter < 5 {
      return false
   } else if counter == 5 {
      return true
   }  else {
      return true
   }
}
```